src/Request/JsonConvertibleParamConverter.php

```diff
@@ -28,6 +28,7 @@
  * ParamConverter that converts JSON objects with underscore notation mapped to snake-cased, public properties of
  * classes that implement JsonConvertible.
  *
+ * @SuppressWarnings(PHPMD.MissingImport)
```

Comment:
Would you be so kind to add some documentation why this is used?

Reply:
Coding standerd changed, don't want to change code when it's not needed